src/dosbox.cpp

```diff
@@ -834,9 +834,8 @@ void DOSBOX_Init(void) {
 	Pstring = Pmulti_remain->GetSection()->Add_string("parameters",Property::Changeable::WhenIdle,"");
 	Pmulti_remain->Set_help("see serial1");
 
-	Pstring = secprop->Add_path("phonebookfile", Property::Changeable::OnlyAtStart, "phonebook-" VERSION ".txt");
-	Pstring->Set_help("File used to map fake phone numbers to addresses.");
-
+	pstring = secprop->Add_path("phonebookfile", only_at_start, "phonebook.txt");
+	pstring->Set_help("File used to map fake phone numbers to addresses.");
```

Comment:
"File used to map fake phone numbers to addresses, formatted as\n"
"PHN HOST:PORT, one entry per line. Example: 5551234 host.org:23"

Reply:
Format of the file is already described in the manual, so I'll keep it as it is.